Rakefile.dist

```diff
@@ -65,6 +66,14 @@ task :install => :build do |t|
 
   # ssh
   mkdir_p "#{DATAROOTDIR}/foreman/.ssh"
+
+  # cron scripts
+  mkdir_p "#{SYSCONFDIR}/cron.hourly"
+  cp_p "script/passenger-recycler.rb", "#{SYSCONFDIR}/cron.hourly/passenger-recycler"
+
+  # configuration files
+  mkdir_p "#{SYSCONFDIR}/foreman"
+  cp_p "script/passenger-recycler.rb.conf.example", "#{SYSCONFDIR}/foreman/passenger-recycler.rb.conf"
```

Comment:
Yeah, I am introducing this as a new config file, but asking in general - I see we carry lots of configuration files in SPEC when this file might be better home for them.